R/rules.R

```diff
@@ -52,12 +52,16 @@ print.rule <- function(x, ...) {
   if (length(x) == 0) {
     cat("Empty mapping.\n")
   } else {
-    for (i in seq_len(length(x))) {
-      cat(nms[i], " <- ", if (length(x[[i]]) > 1) sprintf("[%s]", toString(x[[i]])) else x[[i]], "\n")
+    for (i in seq_along(x)) {
+      ori_nms <- if (length(x[[i]]) > 1) sprintf("[%s]", toString(x[[i]])) else x[[i]]
```

Comment:
is this line makeing a difference? (in the snapshots there seems no `[]` )

Reply:
that seems to be redundant, plus now we only accept characters so I don't think the length can be >1. This is probably a residual from previous design.

Reply:
I think that would make sense if we were printing the rules like that:
```
x <- "a", "b"
```
but currently we are doing:
```
x <- "a"
x <- "b"
```

Should we change ?

Reply:
I think we can update it, this is a bit clearer